### PR TITLE
Removing no longer relevant details from Data Hub Guide pages

### PR DESCRIPTION
**Pull request by ezagroba**

### CODE REVIEW

File: content/en/docs/data-hub/_index.md [hunk 1]
```diff
@@ -13,81 +13,68 @@ cascade:
...
 
 {{% alert color="info" %}}
-The Data Hub integration is available in Studio Pro version [8.14](/releasenotes/studio-pro/8.14/) and above. To use the latest functionality of Data Hub, download the latest version of [Studio Pro](https://marketplace.mendix.com/link/studiopro/). {{% /alert %}}
+For more details about which Data Hub features are included in a Mendix license, see the [Data Hub License Limitations](/refguide/consumed-odata-service-requirements/#license-limitations) section of *Consumed OData Service Requirements*.
+{{% /alert %}}
 
-Follow the journey for building an app using available datasets (or entity sets for services from Mendix apps) from the [Data Hub](https://hub.mendix.com) home page, or try building a quick app by following [How to Share Data Between Apps](/data-hub/share-data/).
+Follow the journey for building an app using available datasets (or entity sets for services from Mendix apps) from the [Data Hub](https://hub.mendix.com) home page, or try building an app by following [How to Share Data Between Apps](/data-hub/share-data/).
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Is the journey building an app or learning how to share data among them?

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Is the journey building an app or learning how to share data among them?

File: content/en/docs/data-hub/_index.md [hunk 2]
```diff
@@ -13,81 +13,68 @@ cascade:
...
-4. Consume registered datasets in Studio Pro to create new apps. 
-5. Upon deployment of an app in Studio Pro, registered services are updated in the Data Hub Catalog to show published dependencies and consumed dependencies, and a connection is established to the shared data sources.
-6. The consuming application directly connects to the source system (as defined in the published service metadata) to use the data associated with the consumed dataset.
-7. The [Data Hub Landscape](/data-hub/data-hub-landscape/) enables users to see the dependencies between components and the relevance of data and to ascertain data lineage.
+1. Published apps and other software expose datasets in OData services. The data is maintained in the respective source apps.
+2. In Studio Pro, exposed OData services (or entity sets as they are named in Studio Pro) are automatically registered in the [Data Hub Catalog](/data-hub/data-hub-catalog/) when your app is deployed. OData services from other applications can also be manually registered in the Data Hub Catalog.
+3. Search in the Data Hub Catalog or Studio Pro to find shared data assets to use.
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: I think I'll keep the reference to the pane here

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: I think I'll keep the reference to the pane here

File: content/en/docs/data-hub/_index.md [hunk 3]
```diff
@@ -13,81 +13,68 @@ cascade:
...
+4. Drag and drop to consume datasets in Studio Pro.  
+5. The consuming application directly connects to the source system (as defined in the published service metadata) to use the data associated with the consumed dataset.
 
 ## 3 Roles in Data Hub{#data-hub-roles}
 
-In Mendix Data Hub, governance is the cornerstone. Curation of shared assets is essential for defining how the shared data assets can be used, identifying the relevance to users, ensuring that the resources can be found by the right users, and managing the accessibility, security and confidentiality levels of the shared data.
+Shared datasets can be curated to ensure that resources can be found by the right users while maintaining their security. The Data Hub Catalog contains only metadata, not data.
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md:39]: OK why aren't we calling it the Metadata Catalog😛

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md:39]: OK why aren't we calling it the Metadata Catalog😛

File: content/en/docs/data-hub/_index.md [hunk 4]
```diff
@@ -13,81 +13,68 @@ cascade:
...
 
 This section describes the general Data Hub user as well as the assigned [Mendix Admin](#admin) and [Data Hub Curator](#curator) roles.
 
 ### 3.1 Data Hub Users
 
-The typical Mendix user is able to use the Data Hub Catalog to search and use registered services. Users can also register new services through published OData services, and as owners, users are able to have curation rights over their OData services.
+The typical Mendix user is able to use the Data Hub Catalog to search and consume registered services. Users can also register new services, and as owners, curate them.
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Remove OData reference?

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Remove OData reference?

File: content/en/docs/data-hub/_index.md [hunk 5]
```diff
@@ -13,81 +13,68 @@ cascade:
...
-{{% alert color="info" %}}
-A user who publishes an OData service is the technical owner of the service and thereby has a [Data Hub Curator](#curator) role over their own services. If they have registered services which are not published and are non-discoverable, only they will be able to find and consume these services. 
-{{% /alert %}}
-
 ### 3.2 Mendix Admin {#admin}
 
 A Mendix Admin can do the following:
 
-* Act as a technical administrator of the organization’s Data Hub
+* Act as a [technical administrator](/developerportal/control-center/data-hub-admin/) of the organization’s Data Hub
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Technical Owner?

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: Technical Owner?

File: content/en/docs/data-hub/_index.md [hunk 6]
```diff
@@ -13,81 +13,68 @@ cascade:
...
+Curators are assigned by the a [Mendix Admin](#admin) and can enrich the metadata of registered services and datasets – for example descriptions, tags, contact information, and discoverability
 
 ## 4 Data Hub Licences {#data-hub-licences}
 
 There are two types of licences for Mendix Data Hub: Data Hub Free and Data Hub Premium.
 
 All Mendix users have access to the Data Hub Free edition. With this, you can retrieve or edit 1000 objects per application per day for each runtime instance when deployed. When that limit is exceeded, you get an error when your applications tries to retrieve more data. For more information, see the [Data Hub License Limitations](/refguide/consumed-odata-service-requirements/#license-limitations) section of *Consumed OData Service Requirements*.
 
-With the Data Hub Premium licence, apps are not limited by any means and can retrieve or edit unlimited objects.
+With the Data Hub Premium licence, apps are not limited. They can retrieve and edit an unlimited number of objects.
```
Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: License (reminder for me to change the spelling all throughout this doc)

Comment (johannahemminger) [inline content/en/docs/data-hub/_index.md]: License (reminder for me to change the spelling all throughout this doc)

File: content/en/docs/data-hub/data-hub-catalog/_index.md [hunk 7]
```diff
@@ -53,13 +53,13 @@ For more details, see [How to Search for Registered Assets](/data-hub/data-hub-c
 
 ### 3.2 Consume – Using Registered Datasets
```
Comment (johannahemminger) [inline content/en/docs/data-hub/data-hub-catalog/_index.md:54]: Maybe put a mention about external entities in this title so people scanning can better make that connection

Comment (johannahemminger) [inline content/en/docs/data-hub/data-hub-catalog/_index.md:54]: Maybe put a mention about external entities in this title so people scanning can better make that connection

File: content/en/docs/data-hub/data-hub-catalog/_index.md [hunk 8]
```diff
@@ -69,35 +69,26 @@ OData services can be registered in the Data Hub in the following ways:
...
 
 	{{% alert color="info" %}}
-  You can filter the search results by environment type in the Data Hub Catalog (**Production**, **Non-production**, and the Mendix Free App environment or **Sandbox**). In the [Data Hub pane](/refguide/data-hub-pane/) of Studio Pro, you can expand search results to include non-production environments. {{% /alert %}}
+  You can filter the search results by environment type in the Data Hub Catalog (**Production**, **Non-production**, and the Mendix Free App environment or **Sandbox**). In the [Data Hub pane](/refguide/data-hub-pane/) of Studio Pro, you can remove the default filter on the search results to include non-production environments. {{% /alert %}}
```
Comment (johannahemminger) [inline content/en/docs/data-hub/data-hub-catalog/_index.md:85]: I heard from Thijs that they were thinking about changing the name from Sandbox to something else, I will check on that

Comment (johannahemminger) [inline content/en/docs/data-hub/data-hub-catalog/_index.md:85]: I heard from Thijs that they were thinking about changing the name from Sandbox to something else, I will check on that

File: content/en/docs/data-hub/_index.md (hunk 1 above)
Comment (ezagroba) [reply content/en/docs/data-hub/_index.md]: It's trying to say the second thing. Maybe this is better? 
```suggestion
Try building an app by following [How to Share Data Between Apps](/data-hub/share-data/).
```

Comment (ezagroba) [reply content/en/docs/data-hub/_index.md]: I got most of your comments on this MR twice. I'm going to close one set.

File: content/en/docs/data-hub/_index.md (hunk 2 above)
Comment (ezagroba) [reply content/en/docs/data-hub/_index.md]: Ok. It's one of those things that's never lived up to the hype. But you're right, it does still exist.

File: content/en/docs/data-hub/_index.md (hunk 3 above)
Comment (ezagroba) [reply content/en/docs/data-hub/_index.md:39]: I KNOW RIGHT.

File: content/en/docs/data-hub/_index.md (hunk 4 above)
Comment (ezagroba) [reply content/en/docs/data-hub/_index.md]: I feel like we say that everywhere, but you're right, never hurts to reiterate. 

```suggestion
The typical Mendix user is able to use the Data Hub Catalog to search and consume registered OData services. Users can also register new services, and as owners, curate them.
```

File: content/en/docs/data-hub/_index.md (hunk 5 above)
Comment (ezagroba) [reply content/en/docs/data-hub/_index.md]: The Mendix Admin (what's being talked about here) is the person who controls your company's Data Hub. A Technical Owner is the person you talk to about a particular service registered in the Catalog. So then let's just say that:
```suggestion
* Act as a [Mendix administrator](/developerportal/control-center/data-hub-admin/) of the organization’s Data Hub
```

File: content/en/docs/data-hub/_index.md (hunk 1 above)
Comment (johannahemminger) [reply content/en/docs/data-hub/_index.md]: Oh wow...that is really annoying, I wonder what happened.

File: content/en/docs/data-hub/data-hub-catalog/_index.md (hunk 7 above)
Comment (ezagroba) [reply content/en/docs/data-hub/data-hub-catalog/_index.md:54]: Added link.

**Review by johannahemminger — APPROVED** [review]

**Pull request merged**